dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumResourceScope.kt

```diff
@@ -180,6 +178,14 @@ internal class RumResourceScope(
         sent = true
     }
 
+    private fun resolveResourceProvider(): ResourceEvent.Provider? {
+        return if (firstPartyHostDetector.isFirstPartyUrl(url)) {
+            ResourceEvent.Provider(domain = url, type = ResourceEvent.Type2.FIRST_PARTY)
```

Comment:
I think the domain is meant to be the host. You can look at how this value is filled to make sure.

Reply:
Yup. I changed that while you were reviewing this :).